whats-new-panel.yaml

```diff
@@ -113,22 +35,59 @@
   targeting: firefoxVersion >= 78
   trigger:
     id: whatsNewPanelOpened
-- id: WHATS_NEW_BADGE_78
+- id: WHATS_NEW_BADGE_80
   content:
     action:
       id: show-whatsnew-button
     badgeDescription:
       string_id: cfr-badge-reader-label-newfeature
-    bucket_id: WHATS_NEW_BADGE_78
+    bucket_id: WHATS_NEW_BADGE_80
     delay: 300000
     target: whats-new-menu-button
   frequency:
     lifetime: 100
   priority: 5
-  targeting: firefoxVersion == 78 && (usesFirefoxSync || hasAccessedFxAPanel || currentDate|date
-    - 2419200000 >= profileAgeCreated) && (!messageImpressions['WHATS_NEW_BADGE_78']
-    || (messageImpressions['WHATS_NEW_BADGE_78']|length >= 1 && currentDate|date -
-    messageImpressions['WHATS_NEW_BADGE_78'][0] <= 345600000)) && 'browser.messaging-system.whatsNewPanel.enabled'|preferenceValue
+  targeting: firefoxVersion == 80 && (usesFirefoxSync || hasAccessedFxAPanel || currentDate|date
+    - 2419200000 >= profileAgeCreated) && (!messageImpressions['WHATS_NEW_BADGE_80']
+    || (messageImpressions['WHATS_NEW_BADGE_80']|length >= 1 && currentDate|date -
+    messageImpressions['WHATS_NEW_BADGE_80'][0] <= 345600000)) && 'browser.messaging-system.whatsNewPanel.enabled'|preferenceValue
   template: toolbar_badge
   trigger:
     id: toolbarBadgeUpdate
+- id: WHATS_NEW_BETTER_PDF_78
+  template: whatsnew_panel_message
+  order: 3
+  content:
+    bucket_id: WHATS_NEW_BETTER_PDF_78
+    published_date: 1593475200000
+    title:
+      string_id: cfr-whatsnew-better-pdf-header
+    body:
+      string_id: cfr-whatsnew-better-pdf-body
+    cta_url: https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/pdf-viewer
+    cta_type: OPEN_URL
+    link_text:
+      string_id: cfr-whatsnew-pip-cta
+  targeting: firefoxVersion >= 78
+  trigger:
+    id: whatsNewPanelOpened
+- id: WHATS_NEW_COOKIES_PROTECTION_80
+  template: whatsnew_panel_message
+  order: 1
+  content:
+    bucket_id: WHATS_NEW_COOKIES_PROTECTION_80
+    published_date: 1598313600000
+    title:
+      string_id: cfr-whatsnew-clear-cookies-header
+    icon_url: chrome://browser/content/logos/tracking-protection.svg
```

Comment:
Just to confirm, it says to use `illustration-cookie-blocked-32.svg` in the doc specs. Did we still want to use the tracking protection one?

Reply:
This is a mistake.